docs/program/apis/cloud.md

```diff
@@ -33,9 +33,7 @@ Cloud app API methods are only available in the Python SDK.
 
 To use the Viam cloud app API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`AppClient`](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient).
 See the following example for reference.
-To find the location secret, go to [Viam app](https://app.viam.com/), and go to the [**Code sample**](https://docs.viam.com/manage/fleet/robots/#code-sample) tab of any of the robots in the location.
-Toggle **Include secret** on and copy the `payload`.
-For the URL, use the address of any of the robots in the location (also found on the **Code sample** tab).
+To find your api key and secret, go to [Viam app](https://app.viam.com/), and navigate to one of your robots' **Security** tab the bottom of your location's page.
```

Comment:
```suggestion
To find your api key and secret, go to [Viam app](https://app.viam.com/), and navigate to one of your robots' **Security** tab.
```
If keeping sentence is confusing, is this what you meant? Robot part API keys in screenshot seems accurate but I can't actually find that in the app at the moment

Reply:
only in staging at the moment  you can see it here: https://github.com/viamrobotics/docs/blob/82453ffbeaf8a0967e23131531bd9116c3b4ffcc/assets/manage/app-usage/robot-secrets.png